js/store/reducers.js

```diff
@@ -32,12 +40,55 @@ function itemsFetching(state) {
 }
 
 function itemsFetched(state, { value }) {
+  var ids = [];
+  const toCache = value.reduce((result, item) => {
+    result[item.id] = createItem(FETCHED, item);
+    ids.push(item.id);
+    return result;
+  }, {});
```

Comment:
I would try not to worry too much about caching in the store - the journey-data library will know more about what caching rules to use, when to cache, when not to, what's stale etc.